.travis/publisher.sh

```diff
@@ -17,19 +21,21 @@ if [ -z ${TRAVIS_TAG+x} ]; then
     exit 1
 fi
 
-echo "---- UPLOAD ARTIFACTS TO GITHUB -----"
-# Download hub
 HUB_VERSION=${HUB_VERSION:-"2.7.0"}
+
+echo "--- Download hub version: $HUB_VERSION ---"
 wget "https://github.com/github/hub/releases/download/v${HUB_VERSION}/hub-linux-amd64-${HUB_VERSION}.tgz" -O "/tmp/hub-linux-amd64-${HUB_VERSION}.tgz"
 tar -C /tmp -xvf "/tmp/hub-linux-amd64-${HUB_VERSION}.tgz"
 export PATH=$PATH:"/tmp/hub-linux-amd64-${HUB_VERSION}/bin"
 
+# TODO: Why?
```

Comment:
Better have an issue or a note for this. We don't put TODOs in the netdata code.